webapp/src/pages/Settings.tsx

```diff
@@ -1,78 +1,694 @@
+  ) => (
+    <Box display="flex" flexDirection="column" paddingTop={1}>
+      <Typography variant="caption">{name}</Typography>
+      {Object.entries(argEntries).map(([field, value], index) => (
+        <Box
+          key={index}
+          display="flex"
+          flexDirection="row"
+          justifyContent="flex-start"
+        >
+          <Typography
+            variant="body2"
+            sx={{
+              s: 1,
+              width: "100%",
+            }}
+          >
+            {field}:
+          </Typography>
+          <Tooltip title={value} placement="bottom">
+            <Typography
+              variant="body2"
+              sx={{
+                s: 1,
+                width: "20ch",
+                whiteSpace: "nowrap",
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+              }}
+            >
+              {value}
+            </Typography>
+          </Tooltip>
+        </Box>
+      ))}
+    </Box>
+  );
+
+  const displayReadonlyFields = (label: string, value: string | undefined) => (
+    <TextField
+      sx={{ m: 1, width: "40ch" }}
+      size="small"
+      variant="standard"
+      label={<Typography fontWeight="bold">{label}</Typography>}
+      value={value}
       disabled={isError || isFetching}
-      onChange={(_, checked) =>
-        setPartialConfig({ ...partialConfig, [field]: checked ? {} : null })
+      InputProps={{
+        readOnly: true,
+        disableUnderline: true,
+        style: { fontSize: 14 },
+      }}
+    />
+  );
+
+  const displayNumberField = (
+    config: SubConfigKeys,
+    field: string,
+    value: number | undefined
+  ) => (
+    <TextField
+      id={field}
+      size="small"
+      label={<Typography fontWeight="bold">{field}</Typography>}
+      type="number"
+      value={value}
+      inputProps={{
+        ...STEPPER[field],
+        style: { fontSize: 14 },
+      }}
+      disabled={!resultingConfig[config]}
+      variant="standard"
+      onChange={(event) =>
+        setPartialConfig({
+          ...partialConfig,
+          [config]: {
+            ...resultingConfig[config],
+            [field]: Number(event.target.value),
+          },
+        })
       }
     />
   );
 
-  return (
-    <Container maxWidth="sm">
-      <FormControl sx={{ width: "100%" }}>
-        <FormGroup>
-          <FormControlLabel
-            control={switchNullOrDefault("behavioral_testing")}
-            label="Perturbation testing"
-          />
+  const displayPostprocessorNumberField = (
+    pipelineIndex: number,
+    pipeline: PipelineDefinition,
+    field: string,
+    postprocessorIdx: number,
+    value: number | undefined
+  ) => (
+    <TextField
+      key={postprocessorIdx}
+      sx={{
+        m: 1,
+        width: "15ch",
+      }}
+      size="small"
+      label={<Typography fontWeight="bold">{field}</Typography>}
+      type="number"
+      value={value}
+      inputProps={{ min: 0, max: 1, step: 0.1 }}
```

Comment:
That's the only input that has the default font size of 16. To make it like the others:
```suggestion
      inputProps={{ min: 0, max: 1, step: 0.1, style: { fontSize: 14 } }}
```

Reply:
Oh yes!. I missed one here. Added the fontsize

Reply:
I resolved this in 8c9d99b1480c423a72af22923af90bbbb4683918